client/mainwindow.cpp

```diff
@@ -201,20 +201,56 @@ void MainWindow::createMenu()
     viewMenu->addSeparator();
 
     auto showEventsMenu = viewMenu->addMenu(tr("&Display in timeline"));
+    addTimelineOptionCheckbox(
+        showEventsMenu,
+        tr("Invite events"),
+        tr("Show invite and withdrawn invitation events"),
+        QStringLiteral("show_invite"),
+        true
+    );
     addTimelineOptionCheckbox(
         showEventsMenu,
         tr("Normal &join/leave events"),
         tr("Show join and leave events"),
         QStringLiteral("show_joinleave"),
         true
     );
+    addTimelineOptionCheckbox(
+        showEventsMenu,
+        tr("Ban events"),
+        tr("Show ban and unban events"),
+        QStringLiteral("show_ban"),
+        true
+    );
+    showEventsMenu->addSeparator();
     addTimelineOptionCheckbox(
         showEventsMenu,
         tr("&Redacted events"),
         tr("Show redacted events in the timeline as 'Redacted'"
            " instead of hiding them entirely"),
         QStringLiteral("show_redacted")
     );
+    addTimelineOptionCheckbox(
+        showEventsMenu,
+        tr("Display na&me changes"),
```

Comment:
```suggestion
        tr("Changes in display na&me"),
```
to avoid the confusion